### ISSUE TITLE
Installation instructions no longer required

### ISSUE THREAD

**Issue by mbaechtold** [4teamwork/ftw.bob.module#1]
I might be wrong, but the installation instructions in https://github.com/4teamwork/ftw.bob.module/blob/master/ftw/bob/module/template/%2Bpackage.fullname%2B/docs/INSTALL.txt are obsolete.

@jone, what do you think?

**Comment by jone** [comment]
Yes, this is standard boilerplate which is missleading. We are not installing thinks as described there.
The `INSTALL.txt` should be removed without a replacement.
